dotnetv3/S3/versioned-examples/DeleteObjectVersionExample/DeleteObjectVersion.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+/// <summary>
+/// This example creates an object in an Amazon Simple Storage Service
+/// (Amazon S3) bucket and then deletes the object version that was
+/// created. The example uses the AWS SDK for .NET version 3.7 and
+/// .NET Core 5.0.
+/// </summary>
+namespace DeleteObjectVersionExample
+{
+    // snippet-start:[S3.dotnetv3.DeleteObjectVersionExample]
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.S3;
+    using Amazon.S3.Model;
+
+    public class DeleteObjectVersion
+    {
+        public static async Task Main()
+        {
+            string bucketName = "doc-example-bucket";
+            string keyName = "verstioned-object.txt";
+
+            // If the AWS region of the default user is different from the AWS
+            // Region of the Amazon S3 bucket, pass the bucket region to the
```

Comment:
pass the bucket region -> pass the AWS Region of the bucket

Reply:
This project is a copy of the project in the containing directory and was removed.